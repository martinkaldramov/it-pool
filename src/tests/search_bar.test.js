import React from 'react';
import { shallow } from 'enzyme';
import SearchBar from '../containers/search_bar';

describe('SearchBar', () => {
  const wrap = shallow(<SearchBar />);

  it('should render correctly', () => {
    expect(wrap).toMatchSnapshot();
  });
});
