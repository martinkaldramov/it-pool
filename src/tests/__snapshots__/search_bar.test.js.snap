// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SearchBar />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <input
          className="search-bar"
          type="text"
/>,
        <FlatButton
          disabled={false}
          fullWidth={false}
          label="Search"
          labelPosition="after"
          labelStyle={Object {}}
          onClick={[Function]}
          onKeyboardFocus={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          onTouchStart={[Function]}
          primary={false}
          secondary={false}
/>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "search-bar",
          "type": "text",
        },
        "ref": null,
        "rendered": null,
        "type": "input",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "disabled": false,
          "fullWidth": false,
          "label": "Search",
          "labelPosition": "after",
          "labelStyle": Object {},
          "onClick": [Function],
          "onKeyboardFocus": [Function],
          "onMouseEnter": [Function],
          "onMouseLeave": [Function],
          "onTouchStart": [Function],
          "primary": false,
          "secondary": false,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": Symbol(react.fragment),
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <input
            className="search-bar"
            type="text"
/>,
          <FlatButton
            disabled={false}
            fullWidth={false}
            label="Search"
            labelPosition="after"
            labelStyle={Object {}}
            onClick={[Function]}
            onKeyboardFocus={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onTouchStart={[Function]}
            primary={false}
            secondary={false}
/>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "search-bar",
            "type": "text",
          },
          "ref": null,
          "rendered": null,
          "type": "input",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "disabled": false,
            "fullWidth": false,
            "label": "Search",
            "labelPosition": "after",
            "labelStyle": Object {},
            "onClick": [Function],
            "onKeyboardFocus": [Function],
            "onMouseEnter": [Function],
            "onMouseLeave": [Function],
            "onTouchStart": [Function],
            "primary": false,
            "secondary": false,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
